app/main/modify_provider/forms.py

```diff
@@ -305,3 +306,13 @@ class ChangeFirmDebtRecoveryForm(ChangeOfficeDebtRecoveryForm):
     title = "Have they have been referred to the Debt Recovery Unit?"
     no_changes_error_message_for_yes_value = "Select no if they are no longer referred to the Debt Recovery Unit. Cancel if you do not want to change the answer."
     no_changes_error_message_for_no_value = "Select yes if they have been referred to the Debt Recovery Unit. Cancel if you do not want to change the answer."
+
+
+class ChangeHoldPaymentsFlagForm(ChangeOfficeHoldPaymentsFlagForm):
+    title = "Do you want to hold payments?"
+    url = "provider/<firm:firm>/hold-payments"
```

Comment:
Can we limit this to advocates and barristers only with `provider/<firm('Barrister','Advocate'):firm>/hold-payments` please.